ui/app/manager/src/pages/page-users.ts

```diff
@@ -735,30 +750,33 @@ export class PageUsers extends Page<AppStateKeyed> {
 
     protected getMQTTSessionTemplate(user: UserModel): TemplateResult {
 
-        const sessionLoader = async () => {
-
-            const userSessionsResponse = await (manager.rest.api.UserResource.getUserSessions(manager.displayRealm, user.id));
-
-            if (!this.responseAndStateOK(() => userSessionsResponse.status === 200, userSessionsResponse, i18next.t("loadFailedUserInfo"))) {
-                return html``;
-            }
-
-            const cols = [i18next.t("address"), i18next.t("since")];
-            const rows = userSessionsResponse.data.map((session) => {
-                return [session.remoteAddress, new Date(session.startTimeMillis)]
-            });
-
-            return html`<or-mwc-table .rows="${rows}" .config="${{stickyFirstColumn:false}}" .columns="${cols}"></or-mwc-table>`;
-        };
+        if (!this._sessionLoader) {
+            this._sessionLoader = this.getSessionLoader(user);
+        }
 
         return html`
             <div id="content" class="panel">
                 <p class="panel-title">${i18next.t('mqttSessions')}</p>
-                ${until(sessionLoader(), html`${i18next.t('loading')}`)}
+                ${until(this._sessionLoader, html`${i18next.t('loading')}`)}
             </div>
         `;
     }
 
+    protected async getSessionLoader(user: UserModel): Promise<TemplateResult> {
+        const userSessionsResponse = await (manager.rest.api.UserResource.getUserSessions(manager.displayRealm, user.id));
+
+        if (!this.responseAndStateOK(() => userSessionsResponse.status === 200, userSessionsResponse, i18next.t("loadFailedUserInfo"))) {
+            return html``;
+        }
+
+        const cols = [i18next.t("address"), i18next.t("since"), ""];
+        const rows = userSessionsResponse.data.map((session) => {
+            return [session.remoteAddress, new Date(session.startTimeMillis), html`<or-mwc-input raised .type="${InputType.BUTTON}" label="${i18next.t("disconnect")}" @or-mwc-input-changed="${() => {this.disconnectSession(user, session)}}"></or-mwc-input>`]
```

Comment:
I think it is better to remove 'raised'. The primary button of the page is the 'save' button.

Reply:
Done